pkg/odo/util/completion/completionhandlers.go

```diff
@@ -39,16 +40,92 @@ var ServiceClassCompletionHandler = func(cmd *cobra.Command, args parsedArgs, co
 	completions = make([]string, 0)
 	services, err := context.Client.GetClusterServiceClasses()
 	if err != nil {
+		complete.Log("error retrieving services")
 		return completions
 	}
 
+	complete.Log(fmt.Sprintf("found %d services", len(services)))
 	for _, class := range services {
 		completions = append(completions, class.Spec.ExternalName)
 	}
 
 	return
 }
 
+// ServicePlanCompletionHandler provides completion for the the plan of a selected service
+var ServicePlanCompletionHandler = func(cmd *cobra.Command, args parsedArgs, context *genericclioptions.Context) (completions []string) {
+	completions = make([]string, 0)
+	if len(args.original.Completed) < 2 {
+		complete.Log("Couldn't extract the service name")
+		return completions
+	}
+
+	inputServiceName := args.original.Completed[1]
+
+	complete.Log(fmt.Sprintf("Using input: serviceName = %s", inputServiceName))
+
+	_, servicePlans, err := service.GetServiceClassAndPlans(context.Client, inputServiceName)
```

Comment:
Actually, maybe use https://github.com/redhat-developer/odo/blob/master/pkg/occlient/occlient.go#L1955 instead?

Reply:
I'll look into it

Reply:
Fixed

Reply:
So I will add the necessary call for the class